prototype.py

```diff
@@ -98,17 +98,17 @@ def scrape(urls):
 
     '''
 
-def chat(message, history):
+def chat(message, state):
     try:
-        # Get location
-        location = get_location()
-        print("Location")
-        print(location)
-
-        # Table storage logic here
-        state = location["region"]
-        # TODO: We need error handling here to ensure that state is in the right format "Michigan" not "MI" etc.  Get from dropdown?
-        state = "Washington"
+        # # Get location
+        # location = get_location()
```

Comment:
Location/State retrieval from zip code is commented as there's state dropdown in chat interface, and that state is what's used for getting eligibility requirement details from Azure storage table & and the same is used in prompt

Reply:
makes sense, thanks!